File: Arrays/Java/Reorganize_The_Array.java
class Complete{
    // Function for finding maximum and value pair
    public static int[] Rearrange (int arr[], int n) {
        // Complete the function
        int[] count = new int[n];
        for (int i = 0; i < n; i++) {
            if(arr[i] > -1)
                count[arr[i]]++;
   	    }
        for(int i = 0; i < count.length; i++) {
            if(count[i] > 0) {
                count[i]=i;
            }
            else {
	        count[i] = -1; 
	    }
        }
        return count;
    }
}

File: Arrays/Python/Reorganize_The_Array.py
def Rearrange (arr,  n) :
    #Complete the function
    i = 0
    while i < n:
        if arr[i] != -1 and arr[i] != i:
            temp = arr[arr[i]]
            arr[arr[i]] = arr[i]
            arr[i] = temp
        else:
            i += 1
    return arr
